daemon/stop.go

```diff
@@ -22,7 +22,7 @@ func (daemon *Daemon) ContainerStop(name string, seconds *int) error {
 		return err
 	}
 	if !container.IsRunning() {
-		err := fmt.Errorf("Container %s is already stopped", name)
+		err := fmt.Errorf("Container %s is not running", name)
 		return errors.NewErrorWithStatusCode(err, http.StatusNotModified)
```

Comment:
maybe we can just use `return errors.NewErrorWithStatusCode(nil, http.StatusNotModified)`